mscore/resourceManager.h

```diff
@@ -15,35 +15,63 @@
 
 #include "ui_resourceManager.h"
 #include "downloadUtils.h"
+#include "plugin/pluginManager.h"
+#include "plugin/pluginUpdater.h"
+#include "plugin/pluginManager.h"
 
 namespace Ms {
 
 class ResourceManager : public QDialog, public Ui::Resource
    {
-    Q_OBJECT
+      Q_OBJECT
```

Comment:
If changing to 6 space indent here, then in the lines below too and also in the brace above

Reply:
Don't know how it was made... Maybe visual studio's IntelliSense and autocomplete. I've reverted it.